src/main/java/setup/cardlists/MakeAllCardsList.java

```diff
@@ -62,19 +71,27 @@ public static void main(String[] args) throws Exception{
 					legalCards.add(card);
 			}
 
+			//Add to an array all cards that were equal or below $1 50% or more of the time
+			List<String> dollarLegalCards = new ArrayList<>();
+			for (String card : timesLegalMap.keySet()) {
```

Comment:
How does this differ from the check for 1c cards? It seems like it's doing the same thing?